pkgs/servers/mail/mailman/default.nix

```diff
@@ -37,6 +57,9 @@ buildPythonPackage rec {
       --replace /usr/sbin/postmap ${postfix}/bin/postmap
     substituteInPlace src/mailman/config/schema.cfg \
       --replace /usr/bin/lynx ${lynx}/bin/lynx
+
+    substituteInPlace setup.py \
+      --replace ",<1.7" ""
```

Comment:
Please include the package name that we don't replace wrong version numbers by accident.